src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxRewriter.cs

```diff
@@ -138,24 +138,24 @@ public virtual SyntaxTrivia VisitTrivia(SyntaxTrivia trivia)
 
         public virtual SyntaxList<TNode> VisitList<TNode>(SyntaxList<TNode> list) where TNode : SyntaxNode
         {
-            SyntaxListBuilder? alternate = null;
+            SyntaxListBuilder<TNode> alternate = default;
```

Comment:
Is this change primarily related to the stated goal of the PR? If not, please consider separating it into its own PR.